corehq/apps/export/models/new.py

```diff
@@ -920,6 +933,53 @@ def get_filters(self):
         return []
 
 
+class SMSExportInstance(ExportInstance):
+    type = SMS_EXPORT
+    identifier = None
+    name = "Messages"
+
+    @classmethod
+    def _new_from_schema(cls, schema):
+        columns = [
+            ("Contact Type", "couch_recipient_doc_type", DOC_TYPE_TRANSFORM),
+            ("Contact ID", "couch_recipient", None),
+            ("Timestamp", "date", None),
+            ("Owner Name", "couch_recipient", OWNER_ID_TRANSFORM),
```

Comment:
Yeah. you're right. I misread the owner id transform. Created a new one https://github.com/dimagi/commcare-hq/pull/14823/commits/6c8b135a70731bfc2c7e650da7767b42cf9b4469#diff-0111f3dfa65bf39568e054504f4430afR23